apps/fluent-tester/src/FluentTester/TestComponents/FocusTrapZone/consts.ts

```diff
@@ -1,2 +1,10 @@
 export const HOMEPAGE_FOCUSTRAPZONE_BUTTON = 'Homepage_FocusTrapZone_Button';
 export const FOCUSTRAPZONE_TESTPAGE = 'FocusTrapZone_TestPage';
+
+/* E2E Testing FocusTrapZone 1 */
+export const FOCUSTRAPZONE_TEST_COMPONENT = 'FocusTrapZone_Test_Component'; // A component on each specific test page
+export const FOCUSTRAPZONE_ACCESSIBILITY_LABEL = 'E2E testing FocusTrapZone_Test_Component accessibility label';
```

Comment:
Do we really use accessibility labels on FTZs?

Reply:
Same question for FocusZone. Not sure how effective the test is if we never encounter the label (I don't believe we get focus on the FocusZone/TrapZone itself, just the focusable items it contains, right?). I don't think FocusZone even supports a11yLabel.

Reply:
Good catch, I agree FZ and FTZ don't need these constants, it doesn't make sense.  I'll remove in my next commit